src/pyhub/llm/agents/tools/code_interpreter/backends/docker.py

```
@@ -0,0 +1,428 @@
+        self.memory_limit = memory_limit
+        self.cpu_quota = cpu_quota
+        self.session_timeout = session_timeout
+        
+        # Initialize Docker client
+        if remote_docker_url:
+            self.client = docker.DockerClient(base_url=remote_docker_url)
+        else:
+            self.client = docker.from_env()
+            
+        # Session manager for persistent state
+        self.session_manager = session_manager or SessionManager(timeout=session_timeout)
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'timeout' in constructor call

(E1123)